packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [21.0.1]
+
+### Fixed
+
+- Resolves transaction custodian promise when set transaction status to submitted or failed ([#3845](https://github.com/MetaMask/core/pull/3845))
+- Fix normalizer ensuring property type is always present in TransactionParams ([#3817](https://github.com/MetaMask/core/pull/3817))
```

Comment:
Should we use backticks around code references such as `type` and `Transactionparams` and the above statuses like `submitted` and `failed`?